WSL/compare-versions.md

```diff
@@ -92,4 +86,16 @@ We recommend that you use WSL 2 as it offers faster performance and 100% system
 - If you rely on a Linux distribution to have an IP address in the same network as your host machine, you may need to set up a workaround in order to run WSL 2. WSL 2 is running as a hyper-v virtual machine. This is a change from the bridged network adapter used in WSL 1, meaning that WSL 2 uses a Network Address Translation (NAT) service for it's virtual network, instead of making it bridged to the host Network Interface Card (NIC) resulting in a unique IP address that will change on restart. To learn more about the issue and workaround that forwards TCP ports of WSL 2 services to the host OS, see [WSL GitHub repository issue 4150, NIC Bridge mode (TCP Workaround)](https://github.com/microsoft/WSL/issues/4150).
 
 > [!NOTE]
-> Consider trying the VS Code [WSL Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to enable you to store your project files on the Linux file system, using Linux command line tools, but also using VS Code on Windows to author, edit, debug, or run your project in an internet browser without any of the performance slow-downs associated with working across the Linux and Windows file systems. [Learn more](tutorials/wsl-vscode.md).
+> Consider trying the VS Code [Remote WSL Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to enable you to store your project files on the Linux file system, using Linux command line tools, but also using VS Code on Windows to author, edit, debug, or run your project in an internet browser without any of the performance slow-downs associated with working across the Linux and Windows file systems. [Learn more](tutorials/wsl-vscode.md).
+
+## WSL in the Microsoft Store and WSL as a Windows component
+
+Originally, WSL has been part of the Windows image itself and you would install it as a Windows component. Since then, the WSL team has [created a preview of WSL in the Microsoft Store](https://devblogs.microsoft.com/commandline/a-preview-of-wsl-in-the-microsoft-store-is-now-available/), which you can [find here](https://aka.ms/wslstorepage). The Store version of WSL is the exact same WSL, with the exact same user experience, the only difference is how often it gets serviced. The Store version of WSL gets serviced in the order of months, while the Windows component version of WSL is serviced in the order of years. We recommend all users move to the Store version of WSL, and future updates will be directed to the Store version of WSL.
```

Comment:
By the time this is live, the app will no longer be preview. Maybe something like WSL has lifted the functionality out of the Windows Image into a package that is available via the Microsoft Store.

Reply:
Hey @benhillis - Does this updated description work? https://github.com/MicrosoftDocs/WSL/pull/1632/commits/9491b8a9db3404ed7d2dfff56eac46feec2e515d